WinUIGallery/Common/FileLoader.cs

```diff
@@ -1,17 +1,14 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
-using System.IO;
 using System.Linq;
-using System.Reflection;
-using System.Text;
 using System.Threading.Tasks;
 using Windows.Storage;
 
 namespace AppUIBasics.Common
 {
     internal class FileLoader
     {
-        public static async Task<string> LoadText(string relativeFilePath)
+        public static async Task<StorageFile> GetJsonFile(string relativeFilePath)
```

Comment:
Why this split in loading the file and reading it as string? Shouldn't it be fine to just load the json as string?

Reply:
you are using `LoadText` inside `LoadLines` and `LoadLines` inside `ListViewPage` so I didn't want to make too many changes.
If we don't use async, all these things will return to the previous state

Reply:
done, i changed async to sync and we dont need to split methods